fv3core/pace/fv3core/initialization/analytic_init.py

```diff
@@ -0,0 +1,64 @@
+from enum import Enum
+
+import pace.util as fv3util
+from pace.fv3core.dycore_state import DycoreState
+from pace.util.grid import GridData
+
+
+class cases(Enum):
+    baroclinic = "baroclinic"
+    tropicalcylclone = "tropicalcyclone"
+
+
+valid_cases = [item.value for item in cases]
```

Comment:
@FlorianDeconinck is good or is there a more elegant/clever/pythonic way to do this?